xline-client/src/types/watch.rs

```diff
@@ -142,55 +144,63 @@ impl WatchRequest {
         self
     }
 
-    /// Set `range_end`
+    /// `range_end` is the end of the range [key, `range_end`) to watch. If `range_end` is not given,
+    /// only the key argument is watched. If `range_end` is equal to '\0', all keys greater than
+    /// or equal to the key argument are watched.
+    /// If the `range_end` is one bit larger than the given key,
+    /// then all keys with the prefix (the given key) will be watched.
     #[inline]
     #[must_use]
     pub fn with_range_end(mut self, range_end: impl Into<Vec<u8>>) -> Self {
         self.inner.range_end = range_end.into();
         self
     }
 
-    /// Set `start_revision`
+    /// Sets the start revision to watch from (inclusive). No `start_revision` is "now".
     #[inline]
     #[must_use]
     pub const fn with_start_revision(mut self, revision: i64) -> Self {
         self.inner.start_revision = revision;
         self
     }
 
-    /// Set `progress_notify`
+    /// `progress_notify` is set so that the Xline server will periodically send a `WatchResponse` with no events to the new watcher if there are no recent events. It is useful when clients wish to recover a disconnected watcher starting from a recent known revision. The xline server may decide how often it will send notifications based on current load.
     #[inline]
     #[must_use]
     pub const fn with_progress_notify(mut self) -> Self {
         self.inner.progress_notify = true;
         self
     }
 
-    /// Set `filters`
+    /// filters filter the events at server side before it sends back to the watcher.
```

Comment:
`filters` filter the events on the server side before it sends back to the watcher. The filter should be decorated with a backtick and the correct expression is "on XX side", not "at XX side"